examples/Cycling leg

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
```

Comment:
Please make a directory called `cycling_leg` and then put this file in it and name the file `run.py`.

Reply:
done